src/stories/Library/Lists/list-find-on-shelf/ListFindOnShelf.tsx

```diff
@@ -0,0 +1,39 @@
+export type ListFindOnShelfProps = {
+  manifestationName: string;
+  location: string;
+  nrAvailable: number;
+  nrOfListItems: number;
+};
+
+const ListFindOnShelf: React.FC<ListFindOnShelfProps> = ({
+  manifestationName,
+  location,
+  nrAvailable,
+  nrOfListItems,
+}) => {
+  const numberArray = new Array(nrOfListItems).fill("item");
+
+  return (
+    <ul className="find-on-shelf">
+      <li className="find-on-shelf__header-row text-small-caption">
+        <span>Materiale</span>
+        <span>Find det på hylden</span>
+        <span>Hjemme</span>
+      </li>
+      {numberArray.map((key) => {
+        return (
+          <li className="find-on-shelf__row text-body-medium-regular" key={key}>
+            <span>{manifestationName}</span>
+            <span>{location}</span>
```

Comment:
Consider whether the location actually consists of multiple DOM elements with decorative separators (implemented in CSS) instead of being a single text string.

Reply:
The data looks like this: 
![image](https://user-images.githubusercontent.com/28546954/190171879-88a71013-cc5d-4915-8f32-6d5d9cad3402.png)
I can see location, sublocation, and even department that could be a part of this string. However, even in this case we can just use the "centered dot" ( · ) character to separate these when implementing the react components. And it will look exactly like the design brief I believe. 
So great point, but I'll be running onwards with this as is I think.